nautobot/core/testing/views.py

```diff
@@ -772,7 +772,9 @@ def test_list_objects_with_constrained_permission(self):
             if hasattr(self.model, "name"):
                 self.assertIn(instance1.name, content, msg=content)
                 self.assertNotIn(instance2.name, content, msg=content)
-            elif hasattr(self.model, "get_absolute_url"):
+            elif (
+                hasattr(self.model, "get_absolute_url") and instance1.get_absolute_url() is not None
+            ):  # TODO fix RelationshipAssociation
```

Comment:
RelationshipAssociation Doesn't play well with this test case so may need some tests overloaded since this and like `DynamicGroupMembers` don't make sense to have their own list vies.